packages/react/kibana_context/root/README.mdx

```diff
@@ -0,0 +1,47 @@
+---
+id: react/context/root
+slug: /react/context/root
+title: React Context - Root
+description: This context provider is used only used by the very base root of Kibana.  Unless you're writing tests, a Storybook, or working in core code, you likely don't need this.
+tags: ['shared-ux', 'react', 'context']
+date: 2023-07-25
+---
+
+## Description
+
+This package contains a root context provider for Kibana rendering.  It handles operations that should only happen _once_ when the browser loads a page.
+
+While it would be safer to isolate this in a `core` package, we need to use it in other contexts-- like Storybook and Jest.
+
+```ts
+import React, { useEffect } from 'react';
+import { BehaviorSubject } from 'rxjs';
+
+import { I18nProvider } from '@kbn/i18n-react';
+import { KibanaRenderContextProvider } from '@kbn/react-kibana-context-render';
```

Comment:
Is this (& and code below) supposed to be `KibanaRootContextProvider`?

Reply:
Yes it is, will fix.